File: API/Entities/AreaOfWork.cs
using System.Collections.Generic;

namespace API.Entities
{
  public class AreaOfWork
  {
    public int Id { get; set; }
    public string Description { get; set; }
    public int ServiceOrder { get; set; }

    public List<Consumable> Consumables { get; set; } = new List<Consumable>();
  }
}

File: client-app/src/app/models/areaOfWork.ts
export class AreaOfWorkFormValues {
  description: string = '';
  serviceOrder: number | undefined = undefined;

  constructor(areaOfWork?: AreaOfWorkFormValues) {
    if (areaOfWork) {
      this.description = areaOfWork.description;
      this.serviceOrder = areaOfWork.serviceOrder;
    }
  }
}

export interface AreaOfWork {
  id: number;
  description: string;
  serviceOrder: number;
}
